packages/uniforms-antd/src/ListAddField.tsx

```diff
@@ -21,6 +21,7 @@ const defaultStyle = { width: '100%' };
 function ListAdd({
   disabled,
   icon = <PlusSquareOutlined />,
+  initialCount = 0,
```

Comment:
I think that we don't need this default, right? Bridges do have fallback anyway.